tools/tune/eq/example_spk_eq.m

```diff
@@ -102,12 +102,12 @@ function example_spk_eq()
 %% With FIR EQ try to flatten frequency response within
 %  1 - 13 kHz frequency band.
 if eq.enable_fir
-        eq.fir_minph = 1;
-	eq.fir_beta = 4;
-	eq.fir_length = 63;
-	eq.fir_autoband = 0;
-	eq.fmin_fir = 900;
-	eq.fmax_fir = 10700;
+    eq.fir_minph = 1;
+    eq.fir_beta = 4;
+    eq.fir_length = 63;
+    eq.fir_autoband = 0;
+    eq.fmin_fir = 900;
+    eq.fmax_fir = 10700;
```

Comment:
Please set up your Matlab to indent by 8, also prefer tab if possible. Unnecessary change.